File: hw/Rodriguez/u2/hw18/Zoo/ec_edu_espe_zoo_model/tiger.py
from .feline import Feline
from datetime import date
from .cage import Cage

class Tiger(Feline):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)

    def feed(self, food):
        raise NotImplementedError("Subclasses should implement this method")

File: hw/ccedeno/u2/HW18ZooInPythonJavaScript/ZooInJavaScript/ec.edu.espe.Zoo.model/Tiger.js
const Feline = require('../ec.edu.espe.Zoo.model/Feline');

class Tiger extends Feline {
  constructor(id, description, dateOnborn, gender, numberOfBones, cage) {
    super(id, description, dateOnborn, gender, numberOfBones, cage);
  }

  feed(food) {
    throw new Error('Method "feed" is not supported yet.');
  }
}

module.exports = Tiger;
